playoutdelayextension.go

```diff
@@ -0,0 +1,47 @@
+package rtp
+
+import (
+	"encoding/binary"
+	"errors"
+)
+
+const (
+	playoutDelayExtensionSize = 3
+	playoutDelayMaxValue      = (1 << 12) - 1
+)
+
+var errPlayoutDelayInvalidValue = errors.New("invalid playout delay value")
+
+// PlayoutDelayExtension is a extension payload format in
+// http://www.webrtc.org/experiments/rtp-hdrext/playout-delay
+// 0                   1                   2                   3
+// 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+// +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+// |  ID   | len=2 |       MIN delay       |       MAX delay       |
+// +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+type PlayoutDelayExtension struct {
+	minDelay, maxDelay uint16
+}
+
+// Marshal serializes the members to buffer
+func (p PlayoutDelayExtension) Marshal() ([]byte, error) {
+	if p.minDelay >= playoutDelayMaxValue || p.maxDelay >= playoutDelayMaxValue {
```

Comment:
I think `playoutDelayMaxValue` itself should be allowed, i.e. this line should use `>` instead of `>=`?

Reply:
Done, thanks for catching that.